tests/e2e/flavors.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+  flavors: ['js'],
+};
```

Comment:
Any way we can use an ES module or won't that work with WebdriverIO configs?

Reply:
the config needs to be cjs to work without transpiling. Probably it could be esm if we set up the whole monorepo as esm, but I don't want to do that until everything is migrated, as there's always some tooling that doesn't fully work with it yet (can think of jest for example)